app/helpers/govuk_chat_promo_helper.rb

```diff
@@ -0,0 +1,14 @@
+module GovukChatPromoHelper
+  GOVUK_CHAT_PROMO_BASE_PATHS = %w[
+    /browse/business
+    /browse/business/business-tax
+    /browse/business/setting-up
+    /browse/tax/self-assessment
+    /set-up-limited-company
+    /set-up-self-employed
```

Comment:
In production this path redirects to https://www.gov.uk/set-up-as-sole-trader, so you should change this path to match. It's still a step-by-step.